### DISCUSSION TITLE
Where are downloads stored?

### DISCUSSION THREAD

**Discussion by mikesellt** [lamarios/clipious#547] [General]
I just started using Clipious, and I can't seem to find on my Android phone where the downloaded videos are stored?  I also can't find the downloaded video on my server.  I'd like to be able to download the video and take it into another app or my PC to edit it, copy to a USB, move to Plex library, etc...  Is there a way to do this, or are the downloaded videos only visible within Clipious itself?

**Comment by lamarios** [comment]
The downloads are only visible within clipious for now. I'm not sure if I will change this in the future. You can use the Seal application for advanced download features.

**Comment by lamarios** [comment]
The last version added an option to copy downloaded videos to your phone's download folder
